src/index.js

```diff
@@ -1,17 +1,19 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
```

Comment:
Do you need the ./index.css file, because you`re using Global Styles? if not, import is not needed and file can be removed from project.